velox/docs/functions/spark/comparison.rst

```diff
@@ -0,0 +1,56 @@
+=====================================
+Comparison Functions
+=====================================
+
+.. function:: between(x, min, max) -> boolean
+
+    Returns true if x is within the specified [min, max] range
+    inclusive. Supports TINYINT, SMALLINT, INTEGER, BIGINT, DOUBLE,
+    REAL types. The types of all arguments must be the same.
+
+.. function:: equalto(x, y) -> boolean
+
+    Returns true if x is equal to y. Supports all scalar types. The
+    types of x and y must be the same.
+
+.. function:: equalnullsafe(x, y) -> boolean
+
+    Returns true if x is equal to y. Supports all scalar types. The
+    types of x and y must be the same. this differs from EqualTo in that
+    it returns true (rather than NULL) if both inputs are NULL,
+    and false (rather than NULL) if one of the input is NULL and the other is not NULL.
+
+.. function:: notequalto(x, y) -> boolean
+
+    Returns true if x is not equal to y. Supports all scalar types. The types
+    of x and y must be the same.
+
+.. function:: lessthan(x, y) -> boolean
+
+    Returns true if x is less than y. Supports all scalar types. The types
+    of x and y must be the same.
+
+.. function:: greaterthan(x, y) -> boolean
+
+    Returns true if x is greater than y. Supports all scalar types. The
+    types of x and y must be the same.
+
+.. function:: lessthanorequal(x, y) -> boolean
+
+    Returns true if x is less than y or x is equal to y. Supports all scalar types. The
+    types of x and y must be the same.
+
+.. function:: greaterthanorequal(x, y) -> boolean
+
+    Returns true if x is greater than y or x is equal to y. Supports all scalar types. The
+    types of x and y must be the same.
+
+.. function:: greatest(value1, value2, ..., valueN) -> [same as input]
+
+    Returns the largest of the provided values. Supports all scalar types. 
+    The types of all arguments must be the same.
+
+.. function:: least(value1, value2, ..., valueN) -> [same as input]
+
+    Returns the smallest of the provided values. Supports all scalar types.
```

Comment:
perhaps, clarify how NULL values are handled. Is null considered largest or smallest?

Reply:
Added example and comment.
skipping null values.
```
SELECT greatest(10, 9, 2, 4, 3); -- 10
SELECT greatest(10, 9, 2, 4, 3, null); -- 10
SELECT least(10, 9, 2, 4, 3); -- 2
SELECT least(10, 9, 2, 4, 3, null); -- 2
```